modules/installation-guide/partials/proc_configuring-workspace-exposure-strategies-using-an-operator.adoc

```diff
@@ -58,30 +82,9 @@ metadata:
 spec:
   server:
     serverExposureStrategy: 'single-host'
-  k8s:
```

Comment:
It's not a valid yaml, is it? Is it done by purpose?

Reply:
nice catch. fixed